aviary/subsystems/propulsion/propeller_performance.py

```diff
@@ -124,7 +251,10 @@ def setup(self):
 
 class PropellerPerformance(om.Group):
     """
-    Computation of propeller thrust coefficient based on Hamilton Standard.
+    Computation of propeller thrust coefficient based on Hamilton Standard or a user
+    provided propeller map. Note that a propeller map allows either helical Mach number or
+    free stream Mach number as input. This infomation will be detected automatically when the 
+    propeller map is loaded into memory.
```

Comment:
this is set by the value of mach_type right?

Reply:
Yes.